Source/WebCore/Modules/websockets/WebSocketHandshake.cpp

```diff
@@ -104,7 +104,7 @@ String WebSocketHandshake::getExpectedWebSocketAccept(const String& secWebSocket
     SHA1 sha1;
     CString keyData = secWebSocketKey.ascii();
     sha1.addBytes(keyData.span());
-    sha1.addBytes(webSocketKeyGUID, std::size(webSocketKeyGUID) - 1);
+    sha1.addBytes(std::span { webSocketKeyGUID, std::size(webSocketKeyGUID) - 1 });
```

Comment:
Maybe we can use this above:
```
constexpr auto webSocketKeyGUID = "258EAFA5-E914-47DA-95CA-C5AB0DC85B11"_span;
```
And then just pass `webSocketKeyGUID ` to `addBytes()` here.

Reply:
Oh, cool! I didn't know that was an option. I'll try that.

Reply:
I couldn't figure out how to make this work without compile errors. I tried numerous approaches, but in the end largely left this unchanged.

Reply:
I don't know what I did wrong before. I tried again and it compiled properly. Weird.